lib/ts/recipe/multifactorauth/types.ts

```diff
@@ -132,6 +154,30 @@ export type RecipeInterface = {
               status: "INPUT_USER_IS_NOT_A_PRIMARY_USER";
           }
     >;
+
+    checkAndCreateMFAContext: (input: {
+        req: BaseRequest;
+        res: BaseResponse;
+        tenantId: string;
+        factorIdInProgress: string;
+        session?: SessionContainerInterface;
+        sessionUser?: User;
+        userAboutToSignIn?: User;
+        userContext: any;
+    }) => Promise<
+        | ({ status: "OK" } & MFAContext)
```

Comment:
why are we returning the context?

Reply:
to be used in createOrUpdateSession post signInUp